src/main/resources/edu/arizona/sista/reach/biogrammar/events/bind_events.yml

```diff
@@ -4,7 +4,7 @@
 - name: binding1a
   label: Binding
   action: mkBinding
-  priority: 4
```

Comment:
Yeah, we really should be using a variable for these priorities.  It's a pain in the ass to change all of these guys.